src/app/contribute/guidelines/component-guidelines/index.html

```diff
@@ -0,0 +1,269 @@
+<stache
+  pageTitle="Component guidelines"
+  showTableOfContents="true"
+  layout="sidebar">
+
+  <stache-page-summary>
+    <p>
+      We highly encourage contributions from all SKY UX users, and we ask that you follow these general guidelines so we can review and accept your contributions in a timely fashion. If you do not follow these guidelines, we cannot accept your contributions.
+    </p>
+  </stache-page-summary>
+
+  <stache-page-anchor>
+    Prerequisites
+  </stache-page-anchor>
+
+  <p>Before you contribute, you must install the <a stacheRouterLink="/learn/get-started/prereqs">SKY UX prerequisites</a>.</p>
+
+  <stache-page-anchor>
+    Contribution guidelines
+  </stache-page-anchor>
+
+  <ul>
+    <li>
+      <b>File an issue before you start any work.</b> The sooner we can discuss potential code changes, the better.
```

Comment:
This can be moved to the general section as it should be the absolute first step for every repo.